python/pythonlab/neighborhood/neighborhood/support/painter_tracker.py

```diff
@@ -0,0 +1,43 @@
+from .position import Position
+from .neighborhood_signal_message import NeighborhoodSignalMessage
+from .neighborhood_signal_key import NeighborhoodSignalKey
+
+NORTH = 'north'
```

Comment:
nit: should we put these constants somewhere so we don't have to duplicate them between this and `direction.py`?

Reply:
Updated!